Makefile

```diff
@@ -111,9 +111,9 @@ endif
 .PHONY: test-cmp-e2e
 test-cmp-e2e:
 ifdef TIMEOUT
-	go test -v github.com/openshift/odo/tests/e2e --ginkgo.focus="odoCmpE2e" -ginkgo.succinct -timeout $(TIMEOUT)
+	go test -v github.com/openshift/odo/tests/e2e --ginkgo.focus="odoCmpE2e" -ginkgo.v -timeout $(TIMEOUT)
```

Comment:
do we still need a verbose flag?
For failed tests, you will see verbose output even with `succinct`

Reply:
Yeah I am having it only until the tests are green I'll revert the last commit that contains this completely before merge